packages/specs/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@eth-optimism/specs",
+  "version": "0.1.0",
+  "license": "MIT",
+  "devDependencies": {
+    "prettier": "2.2.1"
+  },
+  "scripts": {
+    "format": "yarn prettier --write \"{l2geth,protocol}/**/*.md\"",
+    "format:check": "yarn prettier --check \"{l2geth,protocol}/**/*.md\"",
```

Comment:
1. I realize the glob string seems like overkill, but I couldn't get anything else to hit all the md files up and down the tree. 
2. my intention is for `format:check` to run in CI, but I haven't set that up.